src/examples/pagination/CursorDefault.tsx

```diff
@@ -0,0 +1,55 @@
+/**
+ * Copyright Zendesk, Inc.
+ *
+ * Use of this source code is governed under the Apache License, Version 2.0
+ * found at http://www.apache.org/licenses/LICENSE-2.0.
+ */
+
+import React, { useState } from 'react';
+import { CursorPagination } from '@zendeskgarden/react-pagination';
+import { Row, Col } from '@zendeskgarden/react-grid';
+
+const Example = () => {
+  const [cursor, setCursor] = useState(0);
+
+  const pages = [0, 1, 2, 3, 4];
+
+  const onFirst = () => setCursor(0);
+
+  const onLast = () => setCursor(pages.length - 1);
+
+  const onNext = () => {
+    if (cursor < pages.length - 1) {
+      setCursor(cursor + 1);
+    }
+  };
+
+  const onPrevious = () => {
+    if (cursor > 0) {
+      setCursor(cursor - 1);
+    }
+  };
+
+  return (
+    <Row alignItems="center">
+      <Col alignSelf="center">
```

Comment:
`Row` and `Col` are not needed. Pagination components are already center-justified block `nav` elements.